docs/conf.py

```diff
@@ -248,8 +248,9 @@ def generate_r_docs(app):
     source /home/docs/.conda/bin/activate r_env
     export TAR=/bin/tar
     cd {0}
+    git submodule update --init --recursive
```

Comment:
Thanks for the fix! I think it will be better to use config file for such things.
https://docs.readthedocs.io/en/stable/config-file/v2.html#submodules

Could you please try include the following lines 
```
submodules:
  include: all
  recursive: true
```
into
https://github.com/microsoft/LightGBM/blob/master/.readthedocs.yml?

Reply:
P.S. I've enabled RTD builds for the `docs/jlamb` branch and will keep it turned on as you'd asked.

Reply:
sure, I can do that. And thanks for keeping the branch up!

Reply:
> And thanks for keeping the branch up!

![image](https://user-images.githubusercontent.com/25141164/102655108-cadd8080-4182-11eb-9920-0e84ba82c0ee.png)

This will auto-disable RTD builds! 🤣